src/NHibernate/Collection/AbstractPersistentCollection.cs

```diff
@@ -16,7 +16,7 @@ namespace NHibernate.Collection
 	/// Base class for implementing <see cref="IPersistentCollection"/>.
 	/// </summary>
 	[Serializable]
-	public abstract partial class AbstractPersistentCollection : IPersistentCollection
+	public abstract partial class AbstractPersistentCollection : IPersistentCollection, ILazyInitializedCollection
```

Comment:
Another todo here would be great, like `6.0 TODO: remove ILazyInitializedCollection once IPersistentCollection derives from it`.